lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -17,38 +17,13 @@ def define
       multiple_choice :going_or_already_abroad? do
         option :going_abroad
         option :already_abroad
-        save_input_as :going_or_already_abroad
 
-        on_response do
+        on_response do |response|
           self.calculator = uk_benefits_abroad_calculator
-        end
-
-        calculate :country_question_title do
-          if going_or_already_abroad == "going_abroad"
-            "Which country are you moving to?"
-          else
-            "Which country are you living in?"
-          end
-        end
-
-        calculate :why_abroad_question_title do
-          if going_or_already_abroad == "going_abroad"
-            "Why are you going abroad?"
-          else
-            "Why have you gone abroad?"
-          end
-        end
-
-        calculate :going_abroad do
-          going_or_already_abroad == "going_abroad"
-        end
-
-        calculate :already_abroad do
-          going_or_already_abroad == "already_abroad"
-        end
+          calculator.going_abroad = (response == "going_abroad")
 
-        calculate :already_abroad_text_two do
-          " or permanently" if already_abroad
+          self.going_abroad = calculator.going_abroad
+          self.already_abroad = calculator.already_abroad
```

Comment:
```suggestion
```
If we are saving state to the calculator - I don't think we need this. We can just reference the calculator for these attributes.

Reply:
Thank you - I've moved them to the calculator (they are used a lot within the flow - hence me not moving them to calculator initially)